tests/integration/dataset/test_dataset.py

```diff
@@ -35,6 +36,10 @@
 TEST_DATASET_HISTORY_VERSIONS = 10
 
 
+def _assert_frame_equal(df1: pd.DataFrame, df2: pd.DataFrame) -> None:
+    assert_frame_equal(df1, df2, check_dtype=False)
```

Comment:
nit: Can we extract this out into a test helper module for datasets? See it being duplicated in `test_evaluation.py`

Reply:
good idea, just refactored it in [this commit](https://github.com/kolenaIO/kolena/pull/472/commits/56b321868c2cbff0626443ed95e01a95605705fb)